src/types/withStaticProps.ts

```diff
@@ -0,0 +1,4 @@
+import VibeComponent from "./VibeComponent";
+
+export const withStaticProps = <T, P, S>(forwarded: VibeComponent<T, P>, staticProps: S) =>
```

Comment:
I don't really like that we need to wrap it with another higher-order function. Are you sure there isn't a better way?

Reply:
Tried a lot of different stuff, but haven't found anything else that's working